subprojects/zap-clientapi/src/main/java/org/zaproxy/clientapi/core/Alert.java

```diff
@@ -31,6 +31,7 @@
 	public enum Reliability {Suspicious, Warning};
 	public enum Confidence {Low, Medium, High, Confirmed};

```

Comment:
I'll trust yer call. I don't think it would be horrible to add it as part of this. But as long as it isn't lost track of then just think it's fine to seperated....